libraries/AP_HAL_SITL/system.cpp

```diff
@@ -25,7 +25,7 @@ void init()
     gettimeofday(&state.start_time, nullptr);
 }
 
-void WEAK panic(const char *errormsg, ...)
+void panic(const char *errormsg, ...)
```

Comment:
This was actually a bug in master for Cygwin that I needed to patch here so I could compile. This will be fixed in master and will be able to be removed from here.